packages/core/test/collapsible-list/collapsibleListTests.tsx

```diff
@@ -22,7 +22,7 @@ import {
 describe("<CollapsibleList>", () => {
     it("adds className to itself", () => {
         const list = renderCollapsibleList(3, { className: "winner" });
-        assert.lengthOf(list.find(".winner"), 1);
+        assert.isNotEmpty(list.find(".winner"));
```

Comment:
Hmm, this check isn't equivalent. Was this suffering from Enzyme 3's `.hostNodes()` issue (https://github.com/airbnb/enzyme/issues/1174)? Seems like the fix for too many elements here is:

```tsx
assert.lengthOf(list.find(".winner").hostNodes(), 1);
```

Reply:
Yeah but is that really a more meaningful / useful test? IMO this is another tautology and I would rather have a auto-generated test suite for spreading props / classNames rather than including one of these test cases at the start of every component test suite

Reply:
ok I can switch to `hostNodes`

Reply:
would you mind documenting this here? https://palantir.quip.com/QISEA7SVbqfk